geopyspark/avroserializer.py

```diff
@@ -10,13 +9,10 @@
 class AvroSerializer(FramedSerializer):
     def __init__(self,
             schema_json,
```

Comment:
This and the related usages of schema are no longer needed now that we have `avroregistry` to fetch decoders and encoders?